frontend/src/components/SearchCard.js

```diff
@@ -51,31 +50,6 @@ export default class SearchCard extends Component {
     return code;
   };
 
-  courseCard = (provided, snapshot) => {
-    let course = this.props.course;
-    let courseKey = this.props.courseKey;
-    return (
-      <React.Fragment>
-        <CourseCard
-          innerRef={provided.innerRef}
-          draggable={provided.draggableProps}
-          dragHandle={provided.dragHandleProps}
-          course={course}
-          courseKey={courseKey}
-          isDragging={snapshot.isDragging}
-        />
-        {snapshot.isDragging && (
-          <CourseCard
```

Comment:
SearchCards used to use the same CourseCard component as the one used in the schedule. However, I think there are enough differences between the two that it's worth making a different component called SearchCourseCard (which is a CourseCard used only in the search results).